src/BugsnagUnity/Payload/Report.cs

```diff
@@ -62,8 +62,8 @@ public void Ignore()
     /// Gets the metadata associated with this report.
     /// </summary>
     /// <value>The metadata.</value>
-    public Metadata Metadata => Event.Metadata;
     
+    public Dictionary<string, object> Metadata => Event.Metadata;
```

Comment:
This is a breaking change - I don't fully understand why this needs to change to a dictionary.

Reply:
It isn't - the superclass of `Metadata` is `Dictionary<string, object>` and previously there was no added behavior in the Metadata class. The intent was to separate "Metadata class which syncs with the NativeClient" from "instance of metadata attached to a report", however this could instead switch on whether a native client is set on the instance so I'll do that.